config-reload

```diff
@@ -15,16 +15,20 @@ my $libdir = dirname(__FILE__);
 
 our @terms = ();
 
-sub reload_all {
-    my @lines = `urxvt --perl-lib $libdir -pe config-print 2>&1`;
-    my %resource;
-    for (@lines) {
-        $resource{$1} = $2 if (/(.*?): (.*)/);
+sub get_xresources {
+    my @entries = `xrdb -query`;
+    my %resources;
+    for (@entries) {
+        $resources{$2} = $3 if /^(URxvt)?\*([^:]+):\t+(.*)/;
     }
+    return %resources;
```

Comment:
```suggestion
    $resources{borderColor} //= $resources{background};
    $resources{cursorColor} //= $resources{foreground};
    return %resources;
```

While you're at it, you could set the default values as in https://github.com/regnarg/urxvt-config-reload/pull/10.

Reply:
That's a good change but I don't think it would be good git hygiene to bundle it with this PR. I'll certainly add it to my fork though.

Reply:
Ok, I've merged this into my fork.